src/resources/breadCrumbResource.ts

```diff
@@ -0,0 +1,16 @@
+import { BreadcrumbIF } from '@/interfaces'
+
+export const homeBreadCrumb: BreadcrumbIF = {
```

Comment:
I'd be tempted to capitalize these (first char).

Until now, we haven't said that resource objects should be capitalized, but I think that's a good idea since they're not something we change.

Thoughts?

Reply:
Same with the filename.

Reply:
Sounds good, will capitalize 👍

Reply:
In Filings , Create and Edit ui. 
Was updating the version in all the pr's anywho

Reply:
Updated in latest commit

Reply:
Yeah, looks better now. Thx!